packages/wdio-cli/src/types.ts

```diff
@@ -0,0 +1,107 @@
+import { BACKEND_CHOICES, MODE_OPTIONS, REGION_OPTION, COMPILER_OPTION_ANSWERS } from './constants'
```

Comment:
shouldn't the extension of this file be `.d.ts`?

Reply:
There is no reason why it should (or are there any advantages I am missing?). Before the v7 release I will have all types properly be exposed from the `index.d.ts` file.

Reply:
Not quite sure, I supposed `.d.ts` extension is used for type definitions.